### PR TITLE
Move get_started to tutorials and rearrange

### PR DESCRIPTION
**Pull request by lamberta**

### CODE REVIEW

**Review by MarkDaoust — COMMENTED** [review]
I'm not sure we'll get this into r1.9.

The python and cc code may still have @{} doc links pointing back to these and thers's a test that makes sure these work.

 I'm not sure how the release team will feel about touching code files (even if it's comments only) this late in the cycle.

**Comment by lamberta** [comment]
There are surprisingly not many links in the code pointing to the get_started/ material (maybe because this directory was added later).
The ones that I found use the full URL, which our redirects will catch. We should still update those files, but I'll wait until after this all settles.

I've been building the docs regularly and staging. The doc generator did catch a few instances of broken ${} links, but they were all in docs_src/

File: tensorflow/docs_src/tutorials/seq2seq.md [hunk 1]
```diff
@@ -1,5 +0,0 @@
-# Sequence-to-Sequence Models
```
Comment (MarkDaoust) [inline tensorflow/docs_src/tutorials/seq2seq.md]: Are we booting the nmt tutorial? Don't we prefer to include stubs for files that are pulled during site-build?

**Review by MarkDaoust — DISMISSED** [review]

File: tensorflow/docs_src/tutorials/seq2seq.md (hunk 1 above)
Comment (lamberta) [reply tensorflow/docs_src/tutorials/seq2seq.md]: Readded

**Review by MarkDaoust — APPROVED** [review]

**Pull request merged**